From be9f3a97699ffdc82f74de3b676c20adfdd9760b Mon Sep 17 00:00:00 2001
From: Israel Padilla <israel.padilla@intel.com>
Date: Fri, 1 Sep 2017 19:42:25 +0000
Subject: [PATCH] Stateless 2

---
 bin/kafka-server-start.sh | 44 +++++++++++++++++++++++++++++++-------------
 1 file changed, 31 insertions(+), 13 deletions(-)

diff --git a/bin/kafka-server-start.sh b/bin/kafka-server-start.sh
index c6f72c899..d87225171 100755
--- a/bin/kafka-server-start.sh
+++ b/bin/kafka-server-start.sh
@@ -14,10 +14,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-if [ $# -lt 1 ];
+if [ "$1" == "-h" ];
 then
-	echo "USAGE: $0 [-daemon] server.properties [--override property=value]*"
-	exit 1
+	echo "USAGE: $0 [-daemon] [server.properties] [--override property=value]*"
+	exit 0
 fi
 base_dir=$(dirname $0)
 
@@ -34,15 +34,33 @@ if [ "x$KAFKA_HEAP_OPTS" = "x" ]; then
 fi
 
 EXTRA_ARGS=${EXTRA_ARGS-'-name kafkaServer -loggc'}
+CONFIG_FILE=''
 
-COMMAND=$1
-case $COMMAND in
-  -daemon)
-    EXTRA_ARGS="-daemon "$EXTRA_ARGS
-    shift
-    ;;
-  *)
-    ;;
-esac
+while [ $# -gt 0 ]
+do
+    COMMAND=$1
+    case $COMMAND in
+      -daemon)
+        EXTRA_ARGS="-daemon "$EXTRA_ARGS
+        shift
+        ;;
+      *server.properties)
+        CONFIG_FILE="$COMMAND"
+        shift
+        ;;
+      *)
+        break
+        ;;
+    esac
+done
 
-exec $base_dir/kafka-run-class.sh $EXTRA_ARGS kafka.Kafka "$@"
+if [ $CONFIG_FILE ] ; then
+    echo "Config file (User selected): $CONFIG_FILE"
+    exec $base_dir/kafka-run-class.sh $EXTRA_ARGS kafka.Kafka $CONFIG_FILE "$@"
+elif [ -d /etc/kafka -a "$(ls -A /etc/kafka)" ] ; then
+    echo "Config file (/etc/kafka exists): /etc/kafka/server.properties"
+    exec $base_dir/kafka-run-class.sh $EXTRA_ARGS kafka.Kafka /etc/kafka/server.properties "$@"
+else
+    echo "Config file (Clear Linux default): /usr/share/defaults/kafka/server.properties"
+    exec $base_dir/kafka-run-class.sh $EXTRA_ARGS kafka.Kafka /usr/share/defaults/kafka/server.properties "$@"
+fi
-- 
2.14.1

