From df169a97d925050d79f205ecacd6cc1dbb3a5c98 Mon Sep 17 00:00:00 2001
From: Athenas Jimenez Gonzalez <athenas.jimenez.gonzalez@intel.com>
Date: Tue, 22 Aug 2017 23:18:21 +0000
Subject: [PATCH 2/2] Change default log dir

---
 bin/kafka-run-class.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/bin/kafka-run-class.sh b/bin/kafka-run-class.sh
index 5aa96c0..359c140 100755
--- a/bin/kafka-run-class.sh
+++ b/bin/kafka-run-class.sh
@@ -152,7 +152,7 @@ fi
 
 # Log directory to use
 if [ "x$LOG_DIR" = "x" ]; then
-  LOG_DIR="$base_dir/logs"
+  LOG_DIR="/var/log/kafka"
 fi
 
 # Log4j settings
-- 
2.14.1

